recipes/gargammel/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+export CFLAGS="$CFLAGS -I$PREFIX/include"
```

Comment:
This line is not necessary as the CFLAGS as defined by Conda already add $PREFIX/include.